src/EditorFeatures/DiagnosticsTestUtilities/NamingStyles/NamingStylesTestOptionSets.cs

```diff
@@ -3,33 +3,34 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
 using Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
 using Microsoft.CodeAnalysis.NamingStyles;
 using Roslyn.Utilities;
-
-#if CODE_STYLE
-using Microsoft.CodeAnalysis.Internal.Options;
-#else
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Simplification;
-#endif
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.Diagnostics.NamingStyles
 {
+#if !CODE_STYLE
```

Comment:
Do we have code using this in the CODE_STYLE layer despite it's in EditorFeatures? There's still some linked file schenanigans going on or something?

Reply:
This file is only needed for testing using the legacy IDE test framework. For those cases, I have just done an explicit link into CodeStyle layer test utilities project based on IDE test framework. This can go away once we port tests to new test framework.